hooks/useArboStatistics.tsx

```diff
@@ -0,0 +1,24 @@
+import { useQuery } from "@tanstack/react-query";
+import { gql } from "@apollo/client";
+import { request } from 'graphql-request';
+import { ArbovirusDataStatistics } from "@/gql/graphql";
+
+export const arbovirusDataStatistics = gql`
+  query arbovirusDataStatistics {
+    arbovirusDataStatistics {
+        patricipantCount,
+        sourceCount,
+        estimateCount,
+        countryCount
+    }
+  }
+`
+
+export const arbovirusDataStatisticsQueryKey = "arbovirusDataStatistics";
+
+export function useArboDataStatistics() {
+  return useQuery<ArbovirusDataStatistics>({
```

Comment:
```suggestion
  return useQuery<ArbovirusDataStatisticsQuery>({
```
Linking the comment I left on Elias' PR because the exact same logic applies here. https://github.com/serotracker/dashboard/pull/195#discussion_r1510023795

Basically I believe that `ArbovirusDataStatisticsQuery` is the correct type here and that comment I linked above explains why. It's a coincidence that the `ArbovirusDataStatisticsQuery` and the `ArbovirusDataStatistics` types match but as time goes on that might not be true so we're probably better off patching this up now.